getnotes/getnotes.py

```diff
@@ -414,8 +422,65 @@ async def findplayer(self, ctx, *,player: Union[ipaddress.IPv4Address, int, str]
         
         except ModuleNotFoundError:
             return await message.edit(content="`mysql-connector` requirement not found! Please install this requirement using `pip install mysql-connector`.")
-                      
+
+
+    async def dph(self, ctx) -> dict: # Deaths per hour
+        """
+        Calculate a player's deaths per hour
+        """
+        player = key_to_ckey(player)
+
+        try:
+            message = await ctx.send("Calculating DPH....")
+            async with ctx.typing():
+                player = await self.player_search(ctx, ckey=player)
+            
+            if player is None:
+                raise ValueError
+
+            dph = round(player['num_deaths'] / (player['living_time'] / 60), 2)
+
+            await message.edit(content=f"DPH of **{player}**: {dph}", embed=embed)
+
+        except ValueError:
+            return await message.edit(content="Player not found.")
+            
+        except mysql.connector.Error  as err:
+            embed=discord.Embed(title=f"Error finding player DPH", description=f"{format(err)}", color=0xff0000)
+            return await message.edit(content=None,embed=embed)
+        
+        except ModuleNotFoundError:
+            return await message.edit(content="`mysql-connector` requirement not found! Please install this requirement using `pip install mysql-connector`.")
+
+
+    async def nph(self, ctx) -> dict: # Notes per hour
+        """
+        Calculate a player's notes per hour
+        """
+        player = key_to_ckey(player)
+
+        try:
+            message = await ctx.send("Calculating NPH....")
+            async with ctx.typing():
+                player = await self.player_search(ctx, ckey=player)
+            
+            if player is None:
+                raise ValueError
+
+            nph = round(player['notes'] / (player['living_time'] / 60), 2)
```

Comment:
should be encased with a try/except to capture ZeroDivisionError exceptions.
```suggestion
            try:
                nph = round(player['num_deaths'] / (player['living_time'] / 60), 2)
            except ZeroDivisionError:
                nph = 0
```